pkg/machine/qemu/machine.go

```diff
@@ -696,6 +685,35 @@ func (v *MachineVM) Start(name string, _ machine.StartOptions) error {
 	return nil
 }
 
+// propagateHostEnv is here for providing the ability to propagate
+// proxy and SSL settings (e.g. HTTP_PROXY and others) on a start
+// and avoid a need of re-creating/re-initiating a VM
+func propagateHostEnv(cmdLine []string) []string {
+	varsToPropagate := make([]string, 0)
+
+	for k, v := range machine.GetProxyVariables() {
+		varsToPropagate = append(varsToPropagate, fmt.Sprintf("%s=%q", k, v))
+	}
+
+	if sslCertFile, ok := os.LookupEnv("SSL_CERT_FILE"); ok {
+		pathInVM := filepath.Join(machine.UserCertsTargetPath, filepath.Base(sslCertFile))
+		varsToPropagate = append(varsToPropagate, fmt.Sprintf("%s=%q", "SSL_CERT_FILE", pathInVM))
+	}
+
+	if _, ok := os.LookupEnv("SSL_CERT_DIR"); ok {
```

Comment:
Shouldn't this be using $SSL_CERT_DIR

Reply:
Could you explain why you think that? Then it would look up a variable named `$SSL_CERT_DIR`, which is not the intention here.

Reply:
I just think it is weird that the SSL_CERT_FILE environment variable being set, is an on/off indicator to set the variable inside the machine, with no relationship to what the SSL_CERT_FILE is set to.

But I really do not  understand what is going on here. Bottom line the SSL_CERT_FILE HostEnv is not being propagated, from what I see.

Reply:
Ok, now I understand. The value of `SSL_CERT_DIR` is used during ignition: https://github.com/containers/podman/compare/main...bjorndown:podman:feature/pass-ssl-cert-file-via-fw-cfg#diff-2c925d6ff6f9b6848d4c49e021e3ca1cc2641d631b871f1a59279a70fc4314deR505-R508

You are right in that the exact value of `SSL_CERT_DIR` is not being propagated. When it is set on the host it will be set in the VM to wherever the certs were copied during ignition.